src/components/DepositPage.tsx

```diff
@@ -71,15 +82,71 @@ const DepositPage = (props: Props): ReactElement => {
     onConfirmTransaction,
     onToggleDepositWrapped,
   } = props
-
+  const { account, chainId, library } = useActiveWeb3React()
+  const { unstake, amountStaked } = useRewardsHelpers(
```

Comment:
Would you mind renaming these to be minichef-specific since `useRewardsHelpers` will be shared with gauges? Or is that not correct